diffblue.yml

```diff
@@ -1,2 +1,2 @@
-buildCmd: 'gradle clean assemble'
-testCmd: 'gradle test -x compileJava'
+buildCmd: 'mvn compile'
```

Comment:
True, but i'd prefer to strip out the Gradle wrapper completely, as its unnecessary from the point of view of our internal work using the repository etc.